exampleScreenCapturer/src/main/java/com/twilio/video/examples/screencapturer/ScreenCapturerActivity.java

```diff
@@ -68,7 +81,7 @@ public boolean onOptionsItemSelected(MenuItem item) {
         switch (item.getItemId()) {
             case R.id.share_screen_menu_item:
                 String shareScreen = getString(R.string.share_screen);
-
+                screenCapturerManager.nextState();
```

Comment:
I think the use of the state machine pattern works well within the screen capturer service itself, but I think we should provide a more declarative API such as startService/endService so the user has a clearer understanding of what is happening when the button is pressed.

Reply:
That's what I initially had .. then moved to a simpler API approach. I can move back to the more declarative API.